### PR TITLE
github: add issue and pull request templates

### PR DESCRIPTION
**Pull request by kortschak**
Please take a look.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+### What did you do?
+<!-- Please include a link to a minimal reproducer here. -->
+
+
+### What did you expect to happen?
+<!-- Please include an expected ouput image if available. -->
+
+
+### What actually happened?
+<!-- Please include an annotated uput image if available. -->
```
Comment (sbinet) [inline .github/ISSUE_TEMPLATE]: `/uput/output/`

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+-->
+### What are you trying to do?
+
+
+### What did you do?
+<!-- Please include a link to a minimal reproducer here. -->
+
+
+### What did you expect to happen?
+<!-- Please include an expected ouput image if available. -->
```
Comment (sbinet) [inline .github/ISSUE_TEMPLATE]: `/ouput/output/`

Comment (kortschak) [reply .github/ISSUE_TEMPLATE]: Fixed

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (kortschak) [reply .github/ISSUE_TEMPLATE]: Fixed

**Pull request merged**